subprojects/snapshots/src/test/groovy/org/gradle/internal/snapshot/impl/DirectorySnapshotterTest.groovy

```diff
@@ -120,13 +128,23 @@ class DirectorySnapshotterTest extends Specification {
         def excludedFile = rootDir.file("subdir1/a/b/c.html").createFile()
         def notUnderRoot = tmpDir.createDir("root2").file("a.txt").createFile()
         def doesNotExist = rootDir.file("b.txt")
+        // root
+        //   - a.txt <-- rootTextFile
+        //   - a
+        //     - b
+        //       - c.html
+        //       - c.txt <-- nestedTextFile
+        //     - c
+        //       - c.txt  <-- nestedSiblingTextFile
+        //   - subdir1
+        //     - a/b/c.html
```

Comment:
Is this something we want to leave here?

Reply:
You mean the comments about the file structure I added? Yes, I want to leave them there since they help me understand the test.